vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporter.java

```diff
@@ -67,7 +70,10 @@
 
     private Set<String> outputSampleNames;
 
-    public VariantExporter() {
+    private boolean excludeAnnotations;
+
+    public VariantExporter(boolean excludeAnnotations) {
```

Comment:
The previous design intended to allow a `VariantExporter` to be used in multiple export operations, particularly headers and blocks separately (needed for htsget). Thus why the service, the filters and the region query are parameters of `export`.

But then it has an `outputSampleNames` attribute, so that internal status would prevent it from being used in queries over different studies anyway.

So the conclusion of all this comment is that I am fine with adding this new parameter because it also applies both to the headers and the blocks.